astropy/utils/iers/tests/test_iers.py

```diff
@@ -162,8 +162,8 @@ def test_no_auto_download(self):
     def test_simple(self):
         iers_a_file_1 = os.path.join(os.path.dirname(__file__), 'finals2000A-2016-02-30-test')
         iers_a_file_2 = os.path.join(os.path.dirname(__file__), 'finals2000A-2016-04-30-test')
-        iers_a_url_1 = 'file://' + os.path.abspath(iers_a_file_1)
-        iers_a_url_2 = 'file://' + os.path.abspath(iers_a_file_2)
+        iers_a_url_1 = os.path.normpath('file://' + os.path.abspath(iers_a_file_1))
```

Comment:
What does this mean for Windows users? That they always need to use `normpath` for `file://` stuff?

Reply:
In general, all paths should be normalized. However, it doesn't matter as much for regular file system paths since Python is generally able to understand POSIX paths on Windows. The problem in this case is: first of all, we're using a URI, not a normal path; and second of all, that `os.path.join` and `os.path.abspath` already normalized the other parts of the path for Windows, but the `file://` part was not being normalized, so the final product looked something like `file://C:\\Path\\To\\File`, which is nonsense.

Reply:
Short answer to your question: either use `os.path.normpath` on Windows, or users shoudl explicitly spell out `file:\\` when referring to file-based URIs on Windows.